libs/muddle/src/muddle_endpoint_fake.cpp

```diff
@@ -38,7 +38,7 @@ PacketPtr FormatPacket(Packet::Address from, NetworkId const &network, uint16_t
   packet->SetChannel(channel);
   packet->SetMessageNum(counter);
   packet->SetTTL(ttl);
-  packet->SetPayload(std::move(payload));
+  packet->SetPayload(payload);
```

Comment:
A better way IMO to satisfy lint would be to change `Packet::SetPayload()` to accept by value.

Reply:
Perhaps - but that is in Ed's code and I am unwilling to go through more unneccessary lint cycles :)